bin/i18n/i18n_script_utils.rb

```diff
@@ -68,10 +70,21 @@
 
 class I18nScriptUtils
   PROGRESS_BAR_FORMAT = '%t: |%B| %p% %a'.freeze
-  PARALLEL_PROCESSES = Parallel.processor_count / 2
+  PARALLEL_PROCESSES = Parallel.processor_count.freeze
   SOURCE_LOCALE = 'en-US'.freeze
   TTS_LOCALES = (::TextToSpeech::VOICES.keys - %i[en-US]).freeze
 
+  # Checks if the script is running in testing mode
+  def self.testing?
+    # If it's not running an an AWS EC2 instance, it must be a local environment.
+    # And we want to run scripts in testing mode by default on local environment.
+    AWS::EC2.instance_id.nil?
```

Comment:
This seems quite fragile to me. Notably, in addition to baking in some assumptions about the way in which we run our sync which we may not want, this will break in a potentially surprising way for any developers who in fact do use an AWS EC2 instance as their development environment.

I'd strongly recommend introducing a command-line argument or an environment variable rather than attempting this automatic detection. In addition to following the general principle of "explicit is better than implicit", doing so will also make it possible for folks to run the sync locally if so desired (and also if they have correctly set up all necessary credentials locally).